DataConnectors/UniFiSG/Unifi_USG.json

```diff
@@ -0,0 +1,205 @@
+{
+    "id": "UnfiSG",
+    "title": "Unfi Security Gateway (Preview) ",
+    "publisher": "Azure Sentinel Community",
+    "descriptionMarkdown": "The UniFi Security Gateway extends the UniFi Enterprise System to provide cost-effective, reliable routing and advanced security for your network.  Unifi Security Gateway does not output CEF by default, Logstash is used to parse message into CEF for this connector. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "UnifiSG",
+            "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\"\n"
+        },
+        {
+            "metricName":"Total data received",
+            "legend": "UnifiSG-Netflow",
+            "baseQuery": "netflow_CL\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description": "UnifiSG Firewall Logs",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\" and where DeviceEventClassID == \"Firewall\"\n"
+        },
+        {
+            "description": "UnifiSG IPS Logs",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\" and where DeviceEventClassID == \"IPS\"\n"
+        },
+        {
+            "description": "UnifiSG DNS Logs",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\" and where DeviceEventClassID == \"DNS\"\n"
+        },
+        {
+            "description": "UnifiSG DHCP Logs",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\" and where DeviceEventClassID == \"DHCP\"\n"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (UnifiSG)",
+            "lastDataReceivedQuery": "CommonSecurityLog\n| where DeviceVendor == \"Unifi\"\n| where DeviceProduct == \"USG\"\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+        },
+        {
+            "name": "NetFlow (UnifiSG)",
+            "lastDataReceivedQuery":"netflow_CL\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"

```

Comment:
data type change applies here and other places where this is referenced.

Reply:
this is generic netflow.  unifi uses the industry standard netflow formats.